tests/AppAny.HotChocolate.FluentValidation.Tests/SkipValidation.cs

```diff
@@ -29,14 +29,14 @@ public async Task FieldSkipValidation()
           services.AddTransient<IValidator<TestPersonInput>, NotEmptyNameValidator>();
         });
 
-      var result = Assert.IsType<QueryResult>(
+      var result = Assert.IsType<OperationResult>(
         await executor.ExecuteAsync(TestSetup.Mutations.WithEmptyName));
 
-      result.AssertSuceessResult();
+      result.AssertSuccessResult();
```

Comment:
Would you happen to know why the row changed?
I don't see any difference, but maybe it's encoding or endline

Reply:
Suceess => Success for spelling